hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/checksum/TestContainerChecksumTreeManager.java

```diff
@@ -201,7 +201,6 @@ public void testDeletedBlocksPreservedOnTreeWrite() throws Exception {
     assertEquals(metrics.getReadContainerMerkleTreeLatencyNS().lastStat().total(), 0);
     List<Long> expectedBlocksToDelete = Arrays.asList(1L, 2L, 3L);
     checksumManager.markBlocksAsDeleted(container, new ArrayList<>(expectedBlocksToDelete));
-    assertEquals(metrics.getReadContainerMerkleTreeLatencyNS().lastStat().total(), 0);
```

Comment:
Why were these tests removed?

Reply:
We have a similar check below. Now that the read latency is captured for the entire read flow regardless of the present file, this metric gives a non-zero value. It should be fine and this wouldn't skew the metrics.